apps/cowswap-frontend/src/modules/application/containers/TradeWidgetLinks/index.tsx

```diff
@@ -36,6 +39,12 @@ const MENU_ITEMS: MenuItemConfig[] = [
   },
 ]
 
+const TRADE_TYPE_TO_ROUTE: Record<TradeType, string> = {
+  swap: Routes.SWAP,
+  limit: Routes.LIMIT_ORDER,
+  advanced: Routes.ADVANCED_ORDERS,
+}
+
```

Comment:
Doesn't this exist already somewhere else?
If it doesn't, maybe better to put it on `const`?

Reply:
For now we need only here and I don't think we would need it somewhere else